src/tests/test_magic_plot.py

```diff
@@ -469,11 +469,13 @@ def test_bar_two_col(load_data_two_col, ip):
 
 
 @_cleanup_cm()
+@pytest.mark.xfail(reason="Failing intermittently with DuckDB v0.10.0")
```

Comment:
if duckdb is the problem, it'd be better to test with <0.10.0 and open an issue

please open the issue and then document why 0.10.0 is failing

Reply:
Have tagged a couple of issues in the issue description, please check if these are ok